kmip/pie/client.py

```diff
@@ -138,7 +138,8 @@ def close(self):
                 self.logger.exception("could not close client connection", e)
                 raise e
 
-    def create(self, algorithm, length, operation_policy_name=None, name=None):
+    def create(self, algorithm, length, operation_policy_name=None, name=None,
+               cryptographic_usage_mask=list()):
```

Comment:
Let's make the default value for `cryptographic_usage_mask` be `None`. There can be a nasty persistent state bug here if mutable objects are used as default values.